crates/gaiku_baker_modified_marching_cubes/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "gaiku_baker_modified_marching_cubes"
+version = "0.1.0"
+authors = ["Norman Paniagua <normanpaniagua@gmail.com>"]
```

Comment:
You can add yourself as the author too if you want

Reply:
Thanks, didn't want to presume since I am not too bothered either way